blog/2015/03/getting-to-know-katy-huff.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+root: ../../..
+author: Amy Brown
+title: "Getting to Know: Katy Huff"
+date: 2015-03-15
+time: "19:30:00"
+category: ["Community"]
+---
+<!-- start excerpt -->
+<p>
+  This is the first in a series of posts about our contributors. We're posting
+  these so our community can get to know each other better. If you'd like to be
+  profiled, or you'd like to nominate another member, drop us a line.
```

Comment:
@amyrbrown What about replace "drop us a line" with "send a email to communications@lists.software-carpentry.org"?